src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy

```diff
@@ -413,6 +421,7 @@ class FlowExtended {
     }
 
     FlowReroutePayload sync() {
+        log.debug("Sync Flow '$flowId'")
```

Comment:
Very minor. We are using the "present continues" -ing action for the logs, so maybe Sync -> Syncing. But this is up to you

Reply:
sure, will update in my next PR